package_reloader.py

```diff
@@ -14,20 +15,37 @@
     reload_lock = Lock()
 
 
-def casedpath(path):
-    # path on Windows may not be properly cased
-    # https://github.com/randy3k/AutomaticPackageReloader/issues/10
-    r = glob(re.sub(r'([^:/\\])(?=[/\\]|$)', r'[\1]', path))
-    return r and r[0] or path
+if sys.platform.startswith("win"):
+    def realpath(path):
+        # path on Windows may not be properly cased
+        # https://github.com/randy3k/AutomaticPackageReloader/issues/10
+        r = glob(re.sub(r'([^:/\\])(?=[/\\]|$)', r'[\1]', os.path.realpath(path)))
+        return r and r[0] or path
+else:
+    def realpath(path):
+        return os.path.realpath(path)
 
 
 def relative_to_spp(path):
     spp = sublime.packages_path()
-    spp_real = casedpath(os.path.realpath(spp))
-    for p in [path, casedpath(os.path.realpath(path))]:
-        for sp in [spp, spp_real]:
+    spp_real = realpath(spp)
+    for p in {path, realpath(path)}:
```

Comment:
Do we really need to handle the case when path is a symlink pointing inside SPP? I mean, the necessity of going through `realpath(path)` is not quite obvious to me. I know this code has been here before, but.. what's the use case behind that? A symlink in an irrelevant directory pointing to some Sublime plugin module? Sounds quite bizarre..

Reply:
It may be needed if the whole `Packages` directory or `sublime-text-3` directory is symlinked.

Reply:
OK, but then can we go with just `realpath(spp)` and not to check against the `spp` itself? If we don't handle the situation where someone _replaced_ the `sublime-text-3` or `sublime-text-3/Packages` with a symlink to a directory located elsewhere, then AFAIU we can just have it like this:

```
spp = realpath(sublime.package_path())
for p in {path, realpath(path)}:
  if p.startswith(spp + os.sep):
    return p[len(spp):]
```

Reply:
I recall that there are some edge cases where we still need `original_path.startswith(sublime.package_path() + os.sep):`